### PR TITLE
DOCSP-33185 Stop / Resume Sync Job Considerations

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Add stop / resume considerations to stop job page.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-33185/jobs/stopping-jobs/

## JIRA

https://jira.mongodb.org/browse/DOCSP-33185

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=650deb0e492df295cff09272

### CODE REVIEW

**Review by tomhollander — APPROVED** [review]
Thanks for the quick turnaround.

File: source/jobs/stopping-jobs.txt [hunk 1]
```diff
@@ -23,8 +23,12 @@ the type and stage of the sync job:
   partially migrated data remains in the MongoDB database.
 - If you stop a continuous sync job during the CDC stage, the process 
   ends gracefully, and Relational Migrator cleans up any temporary data.
+- Once a sync job is stopped, it cannot be resumed. To migrate data from
```
Comment (jason-price-mongodb) [inline source/jobs/stopping-jobs.txt:26]: "After a ..., you cannot resume it."

**Review by jason-price-mongodb — APPROVED** [review]
@ianf-mongodb LGTM with suggestion. Cheers.

File: source/jobs/stopping-jobs.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/jobs/stopping-jobs.txt]: Updated.

**Pull request merged**